runtime/common/src/claims.rs

```diff
@@ -661,12 +661,14 @@ mod tests {
 	pub struct Test;
 	parameter_types! {
 		pub const BlockHashCount: u32 = 250;
-		pub const MaximumBlockWeight: u32 = 4 * 1024 * 1024;
-		pub const MaximumBlockLength: u32 = 4 * 1024 * 1024;
-		pub const AvailableBlockRatio: Perbill = Perbill::from_percent(75);
+		pub BlockWeights: system::limits::BlockWeights =
```

Comment:
here we can probably ignore all of this and pass a `()` or something that indicates no limit at all, since we don't care at all about weight in any test cases except for one or two pallets in substrate.